### PR TITLE
JsonElement and Utf8JsonReader consistency for TryParse... methods

### LINKED ISSUE #7426: System.Text.Json TryParse... method documentation inconsistent and incomplete
[`JsonElement.TryGetByte()`](https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonelement.trygetbyte?view=net-6.0):
> **Parameters**
> `value` Byte
> When the method returns, contains the byte equivalent of the current JSON number if the conversion succeeded.

[`Utf8JsonReader.TryGetByte()`](https://docs.microsoft.com/en-us/dotnet/api/system.text.json.utf8jsonreader.trygetbyte?view=net-6.0):
> **Parameters**
> `value` Byte
> When this method returns, contains the parsed value.

This is an excerpt from `System.DateTime.TryParse()`:

> When this method returns, contains the DateTime value equivalent to the date and time contained in s, if the conversion succeeded, or MinValue if the conversion failed.

Suggestion to make the description consistent between all methods (`TryGetByte()`, `TryGetInt32()`, ...):

> When this method returns( `true`?), contains the byte equivalent of the current JSON number if the conversion succeeded, or 0 if the conversion failed.

So for each method:
* Change "the method" to "this method"
* Add `true` (?).
* Add the value of the out parameter when the conversion fails.

### PR DESCRIPTION
**Pull request by CodeCasterNL**
## Summary
Update docs to be consistent between JsonElement and Utf8JsonReader.

Fixes #7426

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/dotnet-api-docs?pullRequest=7427) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/dotnet-api-docs?pullRequest=7427) before we can accept your contribution.<br/><br/>:x: CodeCasterNL [sign now](https://cla.dotnetfoundation.org/dotnet/dotnet-api-docs?pullRequest=7427)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/dotnet-api-docs?pullRequest=7427) it.</sub>

File: xml/System.Text.Json/JsonElement.xml [hunk 1]
```diff
@@ -1069,7 +1069,7 @@ For other types, the value of <xref:System.Text.Json.JsonElement.GetRawText> is
         <Parameter Name="value" Type="System.Byte" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="value">When the method returns, contains the byte equivalent of the current JSON number if the conversion succeeded.</param>
+        <param name="value">When this method returns, contains the byte equivalent of the current JSON number if the conversion succeeded, or 0 if the conversion failed.</param>
```
Comment (eiriktsarpalis) [inline xml/System.Text.Json/JsonElement.xml:1072]: Whys is "this" preferable to "the" here? cc @gewarren

Comment (eiriktsarpalis) [reply xml/System.Text.Json/JsonElement.xml:1072]: Oh I see, the methods below all use "this".

**Review by eiriktsarpalis — APPROVED** [review]

**Review by gewarren — APPROVED** [review]
Thank you @CodeCasterNL! This is great.

**Pull request merged**